chore: add project template files (#3)
mdn/project-template: CONTRIBUTOR_ROLES.md

The Community Manager role is distinct in many respects. Still, as the projects are technical, the role shares many of the same requirements and responsibilities as the Maintainer role.

In addition to the responsibilities of a Maintainer, the Community Manager has these additional responsibilities.

- Along with Maintainers, address reports where the code of conduct has been violated and decide on the appropriate action.
- Organize and run community events.
- Determine media strategies to promote the project and onboard new contributors and users.
- Define and implement the onboarding experience.
- Organize community-related project meetings.
- Meet with and ensure a healthy relationship with contributors and partners.
- Assist with issue triage as well as pull request review where appropriate.
- Responsible for monitoring all communication channels, including discussions on GitHub.
- Highlight contributors that have done exceptional work and/or have shown dedication to the project and organization.
- Ensure the health and well-being of the project and all participants.
- Identify and assist with the implementation of automation to improve project sustainability.

> 

Reviewer: Add owner in here - doesn't have to say much, just that it's core team folk

Reviewer: Also I wonder whether this file should be at least linked to (if not mirrored) in contribution docs

## Inactivity

Participants must be active to set an example and show commitment to the project. Conversely, inactivity harms the project as it may lead to unexpected delays, contributor attrition, and a loss of trust in the project.

Inactivity is measured by:

> - Periods of no contributions for longer than two months.
> - Periods of no communication for longer than a month.

Reviewer: ```suggestion
- Periods of no contributions for longer than six months.
- Periods of no communication for longer than three months.
```
I think folks should be able to take a break and come back

# Community participant roles

- [Community participant roles](#community-participant-roles)
  - [Community Participant](#community-participant)
    - [Responsibilities](#responsibilities)

>     - [How community participants can get involved with the community](#how-community-participants-can-get-involved-with-the-community)

Reviewer 2: Maybe two levels for the table of contents is enough information here?

Author: I reckon that makes sense. I have an extension in VSCode that generates this :) I am sure you can configure the level though.

Reviewer 2: I can take this on in a follow-up

### Responsibilities

- Must follow the [Mozilla code of conduct](CODE_OF_CONDUCT.md)

### How community participants can get involved with the community

- Participating in community [discussions](https://github.com/mdn/mdn-community/discussions)
- Helping other users
- Submitting bug reports
- Commenting on issues
- Attending community events

## Contributor

A Contributor contributes directly to the project and adds value to it. Contributions need not be code but can be documentation, research, and helping other community members and contributors. People at the Contributor level may be new contributors or only contribute occasionally.

### Contributor Responsibilities

- Must follow the [Mozilla code of conduct](CODE_OF_CONDUCT.md)
- Follow [contribution guidelines](CONTRIBUTING.md).

### Contributor duties (one or more of the below)

- File issues
- Address issues through [pull requests](CONTRIBUTING.md#pull-request-lifecycle)
- Contributing to documentation

> - Support other community memebers and contributors

Author: ```suggestion
- Support other community members and contributors
```

> - File issues
> - Address issues through [pull requests](CONTRIBUTING.md#pull-request-lifecycle)
> - Contributing to documentation
> - Support other community members and contributors
> - Provide feedback on pull requests
> - Help promote the project
> - Help with community events

Author: (Note from Friday morning session)

```suggestion
- File issues
- Address issues through [pull requests](CONTRIBUTING.md#pull-request-lifecycle)
- Support other community members and contributors
- Help promote the project
```

### Contributor privileges

- Invitation to contributor events.
- Eligibility to become an Organization Member.

## Organization Member

An Organization Member is an established contributor who regularly participates in the project. Organization Members have privileges on the organization level and, as such, are expected to act in the interests of the whole project.

### Requirements to become an Organization Member

Meet one or more of the following requirements on a monthly basis:

> - Two or more accepted pull requests.
> - Reviewed three or more pull requests.
> - Resolved two or more issues.
> - Must have been contributing for at least two months.
> - Must be actively contributing to at least one project area.
> - Must have two sponsors who are also Organization Members.
> - Have two factor authentication enabled for their GitHub account.
> - Have enabled [signed commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

Author: ```suggestion
- Two or more accepted pull requests that resolves two or more issues. (TODO: link to finding an issue)
- Must have been contributing for at least two months.
- Must be actively contributing to at least one project area.
- Have two factor authentication enabled for their GitHub account.
- Have enabled [signed commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
```

> #### Process for nominating an Organization Member
> 
> - The nominator will open a pull request using the appropriate [template](https://github.com/mdn/mdn-community/roles/) against the [MDN Web Docs community repository](https://github.com/mdn/mdn-community/).
> - The nominee will add a comment to the pull request agreeing to all responsibilities of becoming an Organization Member.
> - To be accepted, the pull request needs three approvals. This can be any combination of Organization Members, Invited Experts, or Maintainers.
> - Once the pull request is approved, the new Organization Member is added to the [organization-members team](https://github.com/orgs/mdn/teams/organization-members).

Author: (Note from Friday morning session)

```suggestion
```

In addition to the responsibilities of a Contributor, Organization Members also have the following responsibilities:

### Organization Member responsibilities

> - Continues to contribute regularly, as demonstrated by having at least 24 contributions a year, as indicated by the [insights](https://github.com/orgs/mdn/projects/15/insights) shown on the MDN Motherboard project board.

Author: (Note from Friday morning session)

```suggestion
- Continues to contribute regularly to the MDN Web Docs project.
```

> ### Organization Member privileges
> 
> - May be assigned issues and reviews.
> - Will be added to the [organization-members team](https://github.com/orgs/mdn/teams/organization-members).
> - Can recommend other contributors to become Orginization Members.

Author: (Note from Friday morning session) - Reconsider this once we have set up Holopin.

```suggestion
```

## Invited Experts

Invited Experts are responsible for a specific topic area or project component smaller than an entire project. Invited Experts are automatically assigned for review when pull requests are opened in their topic area. If there is more than one topic expert, reviewers are assigned using a [load-balancing strategy](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-auto-assignment).

> Invited Experts have a track record of contributions, participation, and reviews in the project or organization. They, together with other Reviewers, are responsible for reviewing all changes in their topic area and approving pull requests for merge.

Reviewer: ```suggestion
Invited Experts have a track record of contributions, participation, and reviews, or have proven knowledge in a certain area of expertise. They, together with other Reviewers, are responsible for reviewing changes in their topic area and approving pull requests for merge.
```

### Invited Experts requirements

> - Experience as a Contributor for at least six months.
> - Must be an Organization Member.
> - Has demonstrated an in-depth knowledge of the specific area.
> - Commits to being responsible for their assigned topic area.
> - Is supportive of new and occasional contributors and helps to get pull requests ready to merge.
> - Attends the community meeting which takes place once every two months.

Author: (Note from Friday morning session)

```suggestion
- Has demonstrated an in-depth knowledge of the specific area.
- Commits to being responsible for their assigned topic area.
- Is supportive of new and occasional contributors and helps to get pull requests ready to merge.
- Attends the community meeting which takes place once every two months.
```

#### Process for nominating an Invited Expert

- The nominator will open a pull request using the appropriate [template](https://github.com/mdn/mdn-community/roles/) against the [MDN Web Docs community repository](https://github.com/mdn/mdn-community/).
- The nominee will add a comment to the pull request agreeing to all responsibilities of becoming an Invited Expert.
- To be accepted, the pull request needs three approvals. This can be any combination of Invited Experts, or Maintainers.
- Once the pull request is approved, the new Invited Expert is added to the [invited-experts team](https://github.com/orgs/mdn/teams/invited-experts) and the appropriate topic team.

### Invited experts responsibilities

In addition to the rights and responsibilities of an Organization Member, Invited Experts are responsible for:

> - Following the [review guide](REVIEWING.md).
> - Reviewing most pull requests in their topic area.
> - Helping other contributors become reviewers.

Author: (Note from Friday morning session)

```suggestion
- Following the [review guide](REVIEWING.md).
- Reviewing pull requests in their topic area.
- Helping other contributors become reviewers.
```

### Invited experts privileges

> - Have commit access to the repository.
> - Can recommend and vote for other members to become Organization Members and Invited Experts.
> - Will be added to the Invited Experts and appropriate topic team.

Author: (Note from Friday morning session)

```suggestion
- Have commit access to the repository.
- Can recommend and vote for other members to become Invited Experts.
- Will be added to the Invited Experts and appropriate topic team.
```
